autoscheduler/scraper/tests/helper_function_tests.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+import django.test
+
+from scraper.management.commands.scrape_courses import convert_meeting_time
+from scraper.models.section import generate_meeting_id
+
+class HelperFunctionTests(django.test.TestCase):
+    """ Tests helper functions """
+
+    def test_can_generate_meeting_time(self):
+        """ Tests that scrape_courses.convert_meeting_time can handle a normal time """
+
+        # Arrange/Act
+        time = convert_meeting_time("1230")
+
+        # Assert
+        self.assertEqual(time, datetime.time(12, 30))
+
+    def test_can_generate_null_meeting_time(self):
+        """ Tests that scrape_courses.convert_meeting_time can handle Null times """
+
+        # Arrange/Act
+        time = convert_meeting_time(None)
+
+        # Assert
+        self.assertEqual(time, None)
```

Comment:
Unless this is in a class specifying that's like `ConvertMeetingTimeTests`, I'd generally like to follow the `test_[System Under Test]_[What we're testing]` format, so in this case it would be `test_convert_meeting_time_can_generate_meeting_time` and `test_convert_meeting_time_can_generate_nulll_meeting_time` (although I'd rename them to be not as long).

This is basically so if one of these tests fail, we can very quickly see what is actually failing.